tsdb/querier.go

```diff
@@ -475,13 +475,14 @@ type populateWithDelGenericSeriesIterator struct {
 	// chks are expected to be sorted by minTime and should be related to the same, single series.
 	chks []chunks.Meta
 
-	i         int
-	err       error
-	bufIter   *deletedIterator
-	intervals tombstones.Intervals
+	i          int
+	err        error
+	bufDelIter *deletedIterator
```

Comment:
All these renames make the PR hard to understand, as I can't tell what has actually changed.